### DISCUSSION TITLE
TWO DALY BMS with Influx/Grafana does not show correct values

### DISCUSSION THREAD

**Discussion by cyborgxxl** [jblance/mpp-solar#165] [Q&A] [answered]
I hope someone here can help me. I am using two identical DALY BMS and would like to write the data collected there into an influxdb for Grafana.

My mpp-solar.conf currently looks like this:
```
[SETUP]
pause=10
mqtt_broker=localhost

[DALYBMS1]
model=DALY8S200A
protocol=DALY40
port=/dev/ttyUSB1
baud=9600
command=SOC,cellVoltages,cellTemperatures,cellMinMaxVoltages,cellMinMaxTemps,mosStatus,status
tag=dalybms1
outputs=influx2_mqtt

[DALYBMS2]
model=DALY8S200A
protocol=DALY40
port=/dev/ttyUSB0
baud=9600
command=SOC,cellVoltages,cellTemperatures,cellMinMaxVoltages,cellMinMaxTemps,mosStatus,status
tag=dalybms2
outputs=influx2_mqtt
```

The problem is that in Grafana I cannot select the data separately according to BMS.

So when I call mpp-solar like this: `mpp-solar -C /etc/mpp-solar/mpp-solar.conf --daemon`

Then it looks like this in Grafana:
![grafana](https://user-images.githubusercontent.com/6112343/136023791-282213fa-a3f6-4272-95e0-f179308ec23a.jpg)

Am I doing something wrong here? It works wonderfully with one BMS, but no longer with two.

**Comment by DMIINC** [comment, marked as answer]
You have to select correct Tag in "FROM" line. mpp-solar where = dalybms1

**Reply by cyborgxxl** [reply]
But there is only mpp-solar and nothing else.
![grafana](https://user-images.githubusercontent.com/6112343/136027414-74194ca9-5be9-42e8-87e5-2e74fa0bae65.jpg)

**Reply by cyborgxxl** [reply]
Oh, I'm still a bloody beginner, it seems. I found it. Thank you very much!!!

![grafana](https://user-images.githubusercontent.com/6112343/136027746-aa330b06-72b0-4daa-a976-78c13c885bc0.jpg)

**Comment by DMIINC** [comment]
You are welcome. Where are you using these batteries?

**Reply by cyborgxxl** [reply]
I use these batteries in my 20kwh solar battery array.
